cset_openalex_augmentation_dag.py

```diff
@@ -34,6 +36,29 @@
 args["retries"] = 1
 
 
+def get_updated_version() -> str:
+    """
+    Get the latest semver string for the delivery, and update the corresponding airflow Variable. By default,
+    we bump the minor version with each delivery, but this function allows us to manually set a different
+    part of the semver string to bump in the variable
+    :return: Updated semver string
+    """
+    oa_variable = "cset_openalex_version"
+    version_config = Variable.get(oa_variable, deserialize_json=True)
```

Comment:
It's somewhat confusing having this variable being defined here but its structure doesn't become apparent until lines 56-58, when we save the modified value back to Airtable.  It would be nice if the structure could be made clearer, such as via type definitions (but that may not be particularly easy)

Reply:
I can add a comment